package.json

```
@@ -32,14 +32,14 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand"
   },
   "dependencies": {
-    "@nestjs/config": "^2.2.0",
-    "@nestjs/core": "^9.2.1",
+    "@nestjs/config": "^3.0.0",
+    "@nestjs/core": "^10.1.2",
```

It breaks the compatibility. Coud you add support for both ?
```suggestion
    "@nestjs/core": "^9.2.1 || ^10.1.2",
```